src/store/boards/hooks.tsx

```diff
@@ -43,5 +41,5 @@ export const BoardProvider: FC<{ id: string }> = ({ children, id }) => {
 	);
 };
 
-export const useBoardHooks = (): BoardHooksContext => useContext(boardHooksContext);
-export const useBoard = (): Board => useContext(boardContext);
+export const useBoardHooks = (): BoardHooksContext | undefined => useContext(boardHooksContext);
```

Comment:
should we update also the exposed type and mark it as BC?

Reply:
I am not sure if it is better to expose the possibility to have undefined, or have an assertion inside the hook.
The assertion creates a "logical" requirement (which is already present): the board hooks should be called only inside boards.
Exposing the undefined on the other hand would allow the invocation of the hook from everywhere, even if it is useless.

Reply:
With @CMazzilli we decided that the fix on the type exported will be managed with SHELL-157, which is specific for the cleanup and re-organization of the types